### DISCUSSION TITLE
Request: morisita horn index and clench curve

### DISCUSSION THREAD

**Discussion by iBuitron** [scikit-bio/scikit-bio#2138] [Ideas]
Hi!
There is a way to add :

morisita horn index: https://search.r-project.org/CRAN/refmans/abdiv/html/morisita.html
clench curve https://github.com/EricArcher/sprex/blob/master/R/Clench.R

https://github.com/EricArcher/sprex

thx!

**Comment by qiyunzhu** [comment]
@iBuitron Thanks for the suggestion. Sounds like relevant features. Can you provide some descriptions and literatures to support their relevance to the target audience (like microbial ecologists)?

**Comment by iBuitron** [comment]
@qiyunzhu 

Hi!

I am from Peru, these requirements are based on the Terms of Reference (TOR) of the Ministry of Environment of Peru, for the development of environmental impact studies.

[According to (page 33):](https://cdn.www.gob.pe/uploads/document/file/5787825/5139150-terminos-de-referencia-tdr-comunes-para-los-estudios-de-impacto-ambiental-detallados-categoria-iii-de-proyectos-de-explotacion-beneficio-y-labor-general-mineros-metalicos-a-nivel-de-factibilidad.pdf?v=1706740597) 

![image](https://github.com/user-attachments/assets/6366a60d-423b-4f8e-8941-6bc0a0fb0693)

Translation:
```
c. Description of the biological environment

c.1. Biological Diversity
Biological richness is understood at three levels: ecosystems, species, and genes.

Ecosystems

Characterize terrestrial and aquatic ecosystems.
The evaluation of ecosystems will include:
Characterization of terrestrial and aquatic ecosystems.
Identification and description of existing habitats.
Determination of the richness, abundance, and diversity of species.
Evaluation of the conservation status of existing habitats. In applicable cases, quantitative indicators should be reached, such as: degree of deforestation, overgrazing, fragmentation, soil degradation, among others.
Identification of goods and ecosystem services of habitats and species.
Analysis of the connectivity between habitats and identification of key habitats of ecosystem importance.
Consider aspects or factors that threaten the conservation of habitats.
For the determination of ecological flow, consider the ecological requirements of hydrobiological species, as well as the habitats that depend on this water source.
Species

Evaluation of native, endemic species, key to the ecosystem, migratory species, those in endangered status, as well as those exploited by the local population.
Determination of species richness, abundance, and diversity.
Analysis of diversity through indices such as: Shannon-Wiener, dominance, similarity indices of Jaccard and Sorensen, among others.
Identification of key habitats that warrant protection or some other management measure.
```

[These others are specified in the MINAM Wildlife Guide (Page 7):]( https://www.minam.gob.pe/patrimonio-natural/wp-content/uploads/sites/6/2013/10/GUÃ-A-DE-FAUNA-SILVESTRE.compressed.pdf)

![image](https://github.com/user-attachments/assets/7bac2513-f45a-462f-97b1-543d4a9efd2b)


![image](https://github.com/user-attachments/assets/e906a1e2-abd2-48df-80b2-0dab7150123f)

Thx!

**Comment by qiyunzhu** [comment]
@iBuitron Thanks for providing the background information. We do have an intention to systematically review and upgrade the beta diversity analysis capability in scikit-bio, beyond UniFrac and what scipy's pdist offers. We will gather developer's bandwidth for this. We will keep you updated!
